File: add.c
#include <stdio.h>

int main() {
	float num1,num2,sum,difference,product;

	printf("Enter two numbers");
	scanf("%f %f",&num1,&num2);

	sum=num1+num2;
	difference=num1-num2;
        product=num1*num2

	printf("sum:%.2f\n",sum);
        printf("difference:%.2f\n",difference);
	printf("product:%.2f\n",product);


	return 0;

}

File: add.py
#function to perform basic arithmetic operations
def calculate(a,b):
    add=a+b
    sub=a-b

    return add,sub

#taking input from the user
num1=float(input("enter the first number:"))
num2=float(input("enter the second number:"))

add,sub=calculate(num1,num2)

print(f"add:{add}")
print(f"sub:{sub}")
